tests/requirements.txt

```diff
@@ -9,4 +9,5 @@ pytest-mock==1.6.0
 pytest-watch==3.8.0
 pytest==2.8.5
 requests_mock==1.3.0
+six>=1.11.0
```

Comment:
Why do we need six?

Reply:
I'm guessing it's here because the GCS storage plugin uses it.

Reply:
Maybe we can throw a quick comment - lines beginning with `#` are ignored